main.nf

```diff
@@ -2391,7 +2393,8 @@ process fastgnomad {
 	time '2h'
 
 	when:
-		params.antype == "wgs"
+		// NOTE: The previous change removed myeloid. If we want to remove myeloid, then we don't need the params.run_upd
+		params.run_upd
```

Comment:
The previous version removed myeloid (compared to master). Not sure whether that is what we want?

Reply:
OK no, looking again. This is in the master branch:

```
!params.onco && !params.exome
```

Myeloid is exome, so you are right here, this is wgs only.